### PR TITLE
Level design

### PR DESCRIPTION
**Pull request by JesusRuiz02**

### CODE REVIEW

File: Assets/Scripts/LevelPhysics/Floor.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+
+public class Floor : MonoBehaviour
+{
+   [SerializeField] private float _speed = 1.5f;
+    private bool _dir = true;
+
+    // Update is called once per frame
```
Comment (Horothenic) [inline Assets/Scripts/LevelPhysics/Floor.cs:8]: Remove comment

File: Assets/Scripts/LevelPhysics/Floor.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        if (directionMovement)
+        {
+            transform.Translate(Vector2.right * Time.deltaTime * _speed);
+        }
+        else
+        {
+            transform.Translate(Vector2.left * Time.deltaTime * _speed);
+        }
+
```
Comment (Horothenic) [inline Assets/Scripts/LevelPhysics/Floor.cs:24]: Remove extra line

File: Assets/Scripts/LevelPhysics/Floor.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+   [SerializeField] private float _speed = 1.5f;
+    private bool _dir = true;
+
+    // Update is called once per frame
+    void Update()
+    {
+        Move(_dir);
+    }
+
+    private void Move( bool directionMovement)
```
Comment (Horothenic) [inline Assets/Scripts/LevelPhysics/Floor.cs:14]: Remove space lef of bool

File: Assets/Scripts/LevelPhysics/Floor.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        if (directionMovement)
+        {
+            transform.Translate(Vector2.right * Time.deltaTime * _speed);
+        }
+        else
+        {
+            transform.Translate(Vector2.left * Time.deltaTime * _speed);
```
Comment (Horothenic) [inline Assets/Scripts/LevelPhysics/Floor.cs:22]: Use ternary operator (research if you want) or use a method to abstract this translations into one method

File: Assets/Scripts/LevelPhysics/RotationWheel.cs [hunk 5]
```diff
@@ -0,0 +1,17 @@
+using System;
```
Comment (Horothenic) [inline Assets/Scripts/LevelPhysics/RotationWheel.cs:1]: Remove unused library

File: Assets/Scripts/Player/PlayerMovement.cs [hunk 6]
```diff
@@ -34,11 +34,12 @@ private void PlayerController( bool _moveRight, int speed)
...
-            transform.position = transform.position + new Vector3(-speed * Time.deltaTime, 0, 0); 
+            _rigidbody2D.AddForce(Vector2.left * speed, ForceMode2D.Force);
+            
         }
         else
-        { 
-             transform.position = transform.position +new Vector3(speed * Time.deltaTime, 0, 0); 
+        {
+            _rigidbody2D.AddForce(Vector2.right * speed, ForceMode2D.Force);
         }
```
Comment (Horothenic) [inline Assets/Scripts/Player/PlayerMovement.cs:44]: Remove extra space

**Review by Horothenic — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Player/PlayerMovement.cs [hunk 7]
```diff
@@ -2,6 +2,7 @@
 public class PlayerMovement : MonoBehaviour
 {
     private const string FLOOR = "Floor";
+    private const string MFLOOR = "Mfloor";
```
Comment (Horothenic) [inline Assets/Scripts/Player/PlayerMovement.cs:5]: Same here

File: Assets/Scripts/Player/PlayerMovement.cs [hunk 8]
```diff
@@ -56,10 +56,11 @@ private void DoubleJump()
     
     private void OnCollisionEnter2D(Collision2D collider)
     {
-        if (collider.gameObject.tag == FLOOR)
+        if (collider.gameObject.tag == FLOOR || collider.gameObject.tag == MFLOOR)
         {
             _jumps = _basejumps;
         }
+
```
Comment (Horothenic) [inline Assets/Scripts/Player/PlayerMovement.cs:63]: Remove extra space

**Comment by JesusRuiz02** [comment]
1

**Review by Horothenic — APPROVED** [review]

**Pull request merged**